jbpm-console-ng-process-runtime-forms/jbpm-console-ng-process-runtime-forms-client/src/main/java/org/jbpm/console/ng/pr/forms/client/display/displayers/process/AbstractStartProcessFormDisplayer.java

```diff
@@ -132,6 +144,63 @@ public void init(FormDisplayerConfig<ProcessDefinitionKey> config, Command onClo
 
         initDisplayer();
 
+        final PanelGroup accordion = new PanelGroup();
```

Comment:
@cristianonicolai yeah, that's part of what I told you the other day. If I had more time I would have changed all this part to adapt it to Errai UI.